scipy/linalg/flapack_sym_herm.pyf.src

```diff
@@ -302,6 +302,46 @@ end subroutine <prefix2c>heevd_lwork
    end subroutine <prefix>sytrf_lwork
 
 
+   subroutine <prefix>sytrs(n,nrhs,a,lda,ipiv,b,ldb,info,lower)
+   
+   ! Solve A * X = B for symmetric A matrix after calling ?sytrf
+   
+     callstatement (*f2py_func)((lower?"L":"U"),&n,&nrhs,a,&lda,ipiv,b,&ldb,&info)
+     callprotoargument char*,F_INT*,F_INT*,<ctype>*,F_INT*,F_INT*,<ctype>*,F_INT*,F_INT*
+     
+     <ftype> intent(in), dimension(lda, n), check((lda >= n) && (n >= 0)) :: a
+     <ftype> intent(in,out,copy,out=x),dimension(ldb, nrhs) :: b
+     integer optional,intent(in),check(lower==0||lower==1) :: lower = 0
+     integer intent(in),dimension(n),depend(n) :: ipiv
+     integer intent(hide), depend(a),check(n >= 0) :: n = shape(a, 1)
+     integer intent(hide), depend(b) :: nrhs = shape(b, 1)
+     integer intent(hide), depend(a) :: lda = MAX(1, shape(a, 0))
```

Comment:
Why not check `lda >= n` down here instead of the definition of `a`?

Reply:
They both resolve to same. Doesn't make any difference since f2py collects these and resolves based on `depend` statements.